crates/configuration/src/shared/types.rs

```diff
@@ -355,9 +355,13 @@ impl<'de> de::Visitor<'de> for ArgVisitor {
     where
         E: de::Error,
     {
+        // covers the "-lruntime=debug,parachain=trace" case
+        // TODO: Make this more generic by adding the scenario in the regex below
+        if (v.starts_with("-l") || v.starts_with("-log")) && v.contains(',') {
+            return Ok(Arg::Flag(v.to_string()));
```

Comment:
So basically we are set this as flag instead of traing to parse it right? Did you try if there is no "," like -lparachain=debug ??

Reply:
You are right. This should be better parsed

Reply:
Is ok to treated as flag, but I think maybe there is a bug if isn't contains a ","

Reply:
I tested the case of not containing "," and indeed it breaks; As a result I removed that `||` and retested the possible scenarions and now in both cases a `Flag` (as needed) returns